Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/TMP_KeyboardInputField.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
```

Comment:
Please also move to `Experimental/MixedRealityKeyboard` so all these files are in one location / easier to find.

Reply:
As mentioned above, went for Assets/MixedRealityToolkit.SDK/Experimental/Features/UX